src/pages/Settings/index.js

```diff
@@ -603,7 +699,34 @@ class _AddToHomeScreen extends PureComponent /*:: <Props,State> */ {
 }
 const AddToHomeScreen = connect(undefined, { addToast })(_AddToHomeScreen);
 
-/*:: type SettingsProps = {
+const HashLink = (
+  {
+    firstVisibleTitle,
+    hash,
+    label,
+  } /*: { firstVisibleTitle: string|null, hash:string, label: string } */,
+) => (
+  <li>
+    <Link
+      to={(current) => ({
+        ...current,
+        hash,
+      })}
+      href="#settings-notification"
```

Comment:
Shouldn't it be `href={'#' + hash}`?

Reply:
Oh  I think that actually the error is that `href` that is still there. It's redundant because I'm defining the `to`. 
I'll play a bit with this.